src/sendquery.js

```diff
@@ -140,20 +141,16 @@ async function execute(email, key, project, query, service, params = {}) {
  * get query metadata
  * @param {object} params parameters for substitution into query
  */
-async function queryInfo(params) {
-  const path = params.__ow_path.split('.')[0];
+async function queryInfo(pathname, params) {
+  const [path] = pathname.split('.');
```

Comment:
so a `pathname` will never contain a `.` in a previous segment, or are you only interested in the path *upto* that `.` anyway?

Reply:
And clearer, too. The various substring methods and their respective indices are just too hard to remember.

Reply:
True 😄